broker-cli/commands/wallet.js

```diff
@@ -531,6 +532,46 @@ async function unlock (args, opts, logger) {
   }
 }
 
+/**
+ * Returns all on-chain history of a specific wallet
+ * @param {Object} args
+ * @param {string} args.symbol
+ * @param {Object} opts
+ * @param {string} opts.rpcAddress
+ * @param {Logger} logger
+ * @returns {void}
+ */
+async function history (args, opts, logger) {
+  const { symbol } = args
+  const { rpcAddress } = opts
+
+  try {
+    const client = new BrokerDaemonClient(rpcAddress)
+    const { transactions = [] } = await client.walletService.walletHistory({ symbol })
+
+    if (!transactions) {
```

Comment:
since you initialize `transactions`, this code path will only ever be run if `walletHistory` returns a value like `false` or `0` for transactions.

Reply:
I just want to check the length here. Thanks for catching this